### PR TITLE
Intermediate swaps model for raydium

### PR DESCRIPTION
**Pull request by tarikceric**
Separate model for outputting intermediate swaps performed on Raydium dex

### CODE REVIEW

File: data/testing__swaps_intermediate_raydium.csv [hunk 1]
```diff
@@ -0,0 +1,5 @@
+tx_id,succeeded,swapper,from_mint,from_amt,to_mint,to_amt,swap_index
+3VMB8QpABTaRiVD2PEz2QNdNYqV5nks9dNhq4WBG4HpTDYU7huq7pkj76uvanaJrACq7kMSuypWjsQAUbuCLXjnu,TRUE,7aBDBLE29fkUY99qYY7qY2rgFUDRYGJcEJNWc7B67vPt,So11111111111111111111111111111111111111112,5.2531256,EPjFWdd5AufqSSqeM2qN1xzybapC8G4wEGGkZwyTDt1v,169.997342,1
+4XMga64PPam74iSFPqnXohswejwtdxL1Dj5azTjBfYwE97PfGcP6JSastTwRFHdnzXTcdyRsMwD7t5ZZdpYzjqJj,TRUE,6LnXi13a9j3B4QGRWNxJNWASwZZ2BsgpfXsorNL6xRVD,5idSc21Ht4FTC7jSNe34d6v5FmY8gonswYHpgC7QZCZW,28.18,EPjFWdd5AufqSSqeM2qN1xzybapC8G4wEGGkZwyTDt1v,2.843094,1
+4XMga64PPam74iSFPqnXohswejwtdxL1Dj5azTjBfYwE97PfGcP6JSastTwRFHdnzXTcdyRsMwD7t5ZZdpYzjqJj,TRUE,6LnXi13a9j3B4QGRWNxJNWASwZZ2BsgpfXsorNL6xRVD,EPjFWdd5AufqSSqeM2qN1xzybapC8G4wEGGkZwyTDt1v,2.843094,So11111111111111111111111111111111111111112,0.076159726,2
+4bEAGZx3EkCyuh3arFg1k3hx7CgR2L9aFdxGg3ihWvwdB4KQiheF6JiyRCQcT6LpeJovYdajDg9fsJ3Lor8pN3kk,TRUE,FnJZCfHc6LuZKCWMJGsHmpmqPeVVpxiJR3upiFp8TWsG,CWE8jPTUYhdCTZYWPTe1o5DFqfdjzWKc9WKz6rSjQUdG,0.16553,Es9vMFrzaCERmJfrF4H2FYD4KCoNkY11McCe8BenwNYB,1.307385,1
```
Comment (desmond-hui) [inline data/testing__swaps_intermediate_raydium.csv:5]: Need more samples, this only covers 2 of the listed programs

File: models/silver/swaps/silver__swaps_intermediate_raydium.sql [hunk 2]
```diff
@@ -0,0 +1,399 @@
+{{ config(
+    materialized = 'incremental',
+    unique_key = ["block_id","tx_id","swap_index"],
+    merge_predicates = ["DBT_INTERNAL_DEST.block_timestamp::date >= LEAST(current_date-7,(select min(block_timestamp)::date from {{ this }}__dbt_tmp))"],
+    cluster_by = ['block_timestamp::DATE','_inserted_timestamp::DATE'],
+    post_hook = "ALTER TABLE {{ this }} ADD SEARCH OPTIMIZATION"
```
Comment (desmond-hui) [inline models/silver/swaps/silver__swaps_intermediate_raydium.sql:6]: remove post hook

File: models/silver/swaps/silver__swaps_intermediate_raydium.sql [hunk 3]
```diff
@@ -0,0 +1,399 @@
...
+            OR (
+                program_id = '5quBtoiQqxF9Jv6KYKctB59NT3gtJD2Y65kdnB1Uev3h'
+                AND instruction :accounts [1] :: STRING = '2EXiumdi14E9b8Fy62QcA5Uh6WdHS2b38wtSxp72Mibj'
+            )
+            OR program_id IN (
+                '93BgeoLHo5AdNbpqy9bD12dtfxtA5M2fh3rj72bE35Y3',
+                'routeUGWgWzqBWFcrCfv8tritsqukccJPu3q5GPP3xS'
+            )
+        )
+        AND inner_instruction_program_ids [0] <> 'DecZY86MU5Gj7kppfUCEmd4LbXXuyZH1yHaP2NTqdiZB' --associated with wrapping of tokens
```
Comment (desmond-hui) [inline models/silver/swaps/silver__swaps_intermediate_raydium.sql:71]: I believe you said this condition was for jupiter and saber only?  If so, please remove

**Review by desmond-hui — CHANGES_REQUESTED** [review]

File: models/silver/swaps/silver__swaps_intermediate_raydium.sql (hunk 2 above)
Comment (tarikceric) [reply models/silver/swaps/silver__swaps_intermediate_raydium.sql]: Removed

File: models/silver/swaps/silver__swaps_intermediate_raydium.sql (hunk 3 above)
Comment (tarikceric) [reply models/silver/swaps/silver__swaps_intermediate_raydium.sql]: You are correct - condition removed

File: data/testing__swaps_intermediate_raydium.csv (hunk 1 above)
Comment (tarikceric) [reply data/testing__swaps_intermediate_raydium.csv:5]: 2 examples added each for program id '93BgeoLHo5AdNbpqy9bD12dtfxtA5M2fh3rj72bE35Y3' and '5quBtoiQqxF9Jv6KYKctB59NT3gtJD2Y65kdnB1Uev3h'

**Pull request closed without merging**